nx/lib/nx.ex

```diff
@@ -6658,6 +6658,217 @@ defmodule Nx do
     impl!(tensor).cholesky(out, tensor)
   end
 
+  @doc """
+  Calculates the p-norm of a tensor.
+
+  For the 0-norm, the norm is the number of non-zero elements in the tensor.
+
```

Comment:
Please add a description of the supported options. Note that if `:fro` and `:nuc` are abbreviations, prefer to use full words instead. :)

Reply:
Sure! I forgot to update the docs after adding said options. I'll improve naming as well.